rdagent/utils/env.py

```diff
@@ -439,6 +489,61 @@ class DockerConf(EnvConf):
     retry_wait_seconds: int = 10  # retry wait seconds for the docker run
 
 
+class QlibCondaConf(CondaConf):
+    conda_env_name: str = "qlib4rdagent"
+    enable_cache: bool = False
+    default_entry: str = "qrun conf.yaml"
+    # extra_volumes: dict = {str(Path("~/.qlib/").expanduser().resolve().absolute()): "/root/.qlib/"}
+
+    @model_validator(mode="after")
+    def ensure_qlib_env(self, **data: Any) -> "QlibCondaConf":
```

Comment:
Why not use the function prepare? if it does not work, we may use some mechanism to run once.